gallery/src/styles/NxList/NxListMultiLineExample.html

```diff
@@ -6,27 +6,35 @@
     distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
 
 -->
-<div class="nx-list">
-  <h4 class="nx-list__title">
-    List title
-  </h4>
-  <ul class="nx-list">
-    <li class="nx-list__item">
-      Item 1
-      <p class="nx-list__subtext">This is a second line.</p>
-    </li>
-    <li class="nx-list__item">
-      Item 2
-      <p class="nx-list__subtext">
-        This is a second line. It includes text that might relate to the top line or might not.
-      </p>
-    </li>
-    <li class="nx-list__item">
-      Item 3
-      <p class="nx-list__subtext">
-        A long line of subtext that wraps. Add <code class="nx-code">.nx-truncate-ellipsis</code>
-        to the <code class="nx-code">&lt;p&gt;</code> tag if you don't want the text to wrap.
-      </p>
-    </li>
-  </ul>
-</div>
+<h3 class="nx-h3">
+  Examples of wrapping and truncation in lists
+</h3>
+<ul class="nx-list">
+  <li class="nx-list__item">
+    <span class="nx-list__text">
+      Text wrapping - text that is very long and will wrap onto the next line, standard list item text that is very
+      long and will wrap onto the next line
```

Comment:
None of the examples that I see in zeplin have long content in the field, so I can't know whether it's supposed to wrap or truncate.  I would've expected it to truncate, personally

Reply:
> None of the examples that I see in zeplin have long content in the field, so I can't know whether it's supposed to wrap or truncate. I would've expected it to truncate, personally

I do see now that you've covered how to do that in the next example, but I think the question still stands of what it should do by default.

Reply:
> > None of the examples that I see in zeplin have long content in the field, so I can't know whether it's supposed to wrap or truncate. I would've expected it to truncate, personally
> 
> I do see now that you've covered how to do that in the next example, but I think the question still stands of what it should do by default.

By default is wrap, since truncation requires adding a class. I'd argue that's best since reversing the truncation rules to get wrapping is harder than adding the truncation class

Reply:
@dturner32 ^^